javalin/src/main/java/io/javalin/http/Context.kt

```diff
@@ -276,6 +285,12 @@ interface Context {
     /** Returns true if request is multipart/form-data. */
     fun isMultipartFormData(): Boolean = header(Header.CONTENT_TYPE)?.lowercase(Locale.ROOT)?.contains("multipart/form-data") == true
 
+    /** Returns true if request is application/x-www-form-urlencoded. */
+    fun isFormUrlencoded(): Boolean = header(Header.CONTENT_TYPE)?.lowercase(Locale.ROOT)?.contains("application/x-www-form-urlencoded") == true
+
+    /** Returns true if request is application/x-www-form-urlencoded. */
```

Comment:
```suggestion
    /** Returns true if request is application/json. */
```

I do however wonder if you should use `startsWith`  for each value in the collection as usually content types can have definitions of character encoding etc in it, but it however should start with application. ie application/json; charset=iso-8859-1